tests/unit/test-cmd/test.run.js

```diff
@@ -127,6 +127,34 @@ describe('run', () => {
     });
   });
 
+  it('passes single url parameter to Firefox when specified', () => {
+    const cmd = prepareRun();
+    const {firefoxApp} = cmd.options;
+    const expectedBinaryArgs = ['--url', 'www.example.com'];
+
+    return cmd.run({startUrl: 'www.example.com'}).then(() => {
+      assert.equal(firefoxApp.run.called, true);
```

Comment:
there is a shortcut for this which is a bit more concise: `assert.ok(firefoxApp.run.called)`